okhttp/src/main/kotlin/okhttp3/internal/http2/Http2Reader.kt

```diff
@@ -65,7 +65,7 @@ class Http2Reader(
       source = continuation,
       headerTableSizeSetting = 4096
   )
-
+  private const val EARLY_HINTS_CODE = "103"
```

Comment:
This needs to be a ByteString.  

```suggestion
  private const val EARLY_HINTS_CODE = "103"
```